packages/components/input-number/__tests__/input-number.test.tsx

```diff
@@ -94,6 +94,20 @@ describe('InputNumber.vue', () => {
     expect(wrapper.find('input').element.value).toEqual('4')
   })
 
+  test('value decimals miss prop precision', async () => {
+    const num = ref(0.2)
+    const wrapper = mount(() => <InputNumber step={0.1} v-model={num.value} />)
+    const elInputNumber = wrapper.findComponent({ name: 'ElInputNumber' }).vm
```

Comment:
## Suggest

```ts
const elInputNumber = wrapper.findComponent(InputNumber).vm
```

I think this could be a little more concise!

Reply:
ok soga